Makefile

```diff
@@ -1058,7 +1058,9 @@ release-binary: $(RELEASE_DIR)
 
 .PHONY: release-staging
 release-staging: ## Build and push container images to the staging bucket
-	REGISTRY=$(STAGING_REGISTRY) $(MAKE) docker-build-all docker-push-all release-alias-tag
+	REGISTRY=$(STAGING_REGISTRY) $(MAKE) docker-build-all
+	REGISTRY=$(STAGING_REGISTRY) $(MAKE) docker-push-all
```

Comment:
am I correct to understand that `docker-push-all` and `docker-build-all` won't and can't run in parallel (because push depends on build)? if so which parts are parallelized by using the `-j` option? Is it the docker build commands themselves?